samples/join-team-using-qr-code/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
+  "version": "1.0.0",
+  "id": "{{Microsoft-App-Id}}",
+  "packageName": "com.microsoft.teams.qrjointeam",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "Join team by QR",
+    "full": "Join team by QR"
+  },
+  "description": {
+    "short": "Join team by QR",
+    "full": "This sample demos joining a team using QR code containing the team id."
```

Comment:
This sample demos a feature where user can join a team using QR code containing the team's id.